src/parkapi_sources/converters/opendata_swiss/models.py

```diff
@@ -0,0 +1,201 @@
+    DataclassValidator,
+    EnumValidator,
+    IntegerValidator,
+    ListValidator,
+    Noneable,
+    StringValidator,
+    TimeFormat,
+    TimeValidator,
+)
+
+from parkapi_sources.converters.base_converter.pull import GeojsonFeatureGeometryInput
+from parkapi_sources.models import RealtimeParkingSiteInput, StaticParkingSiteInput
+from parkapi_sources.models.enums import ParkAndRideType, ParkingSiteType, PurposeType
+
+
+@validataclass
+class OpenDataSwissAddressInput:
+    addressLine: str = StringValidator()
+    city: str = StringValidator()
+    postalCode: str = StringValidator()
+
+
+class OpenDataSwissCapacityCategoryType(Enum):
+    STANDARD = 'STANDARD'
+    DISABLED_PARKING_SPACE = 'DISABLED_PARKING_SPACE'
+    RESERVABLE_PARKING_SPACE = 'RESERVABLE_PARKING_SPACE'
+    WITH_CHARGING_STATION = 'WITH_CHARGING_STATION'
+
+
+@validataclass
+class OpenDataSwissCapacitiesInput:
+    categoryType: OpenDataSwissCapacityCategoryType = EnumValidator(OpenDataSwissCapacityCategoryType)
+    total: int = IntegerValidator()
+
+
+@validataclass
+class OpenDataSwissAdditionalInformationInput:
+    de: Optional[str] = Noneable(StringValidator(multiline=False, unsafe=True))
+    en: Optional[str] = Noneable(StringValidator(multiline=False, unsafe=True))
+    it: Optional[str] = Noneable(StringValidator(multiline=False, unsafe=True))
+    fr: Optional[str] = Noneable(StringValidator(multiline=False, unsafe=True))
+
+
+class OpenDataSwissOperationTimeDaysOfWeek(Enum):
+    MONDAY = 'MONDAY'
+    TUESDAY = 'TUESDAY'
+    WEDNESDAY = 'WEDNESDAY'
+    THURSDAY = 'THURSDAY'
+    FRIDAY = 'FRIDAY'
+    SATURDAY = 'SATURDAY'
+    SUNDAY = 'SUNDAY'
+
+    def to_osm_opening_day_format(self) -> str:
+        return {
+            self.MONDAY: 'Mo',
+            self.TUESDAY: 'Tu',
+            self.WEDNESDAY: 'We',
+            self.THURSDAY: 'Th',
+            self.FRIDAY: 'Fr',
+            self.SATURDAY: 'Sa',
+            self.SUNDAY: 'Su',
+        }.get(self, None)
+
+
+@validataclass
+class OpenDataSwissOperationTimeInput:
+    operatingFrom: Optional[time] = Noneable(TimeValidator(time_format=TimeFormat.WITH_SECONDS))
+    operatingTo: Optional[time] = Noneable(TimeValidator(time_format=TimeFormat.WITH_SECONDS))
+    daysOfWeek: Optional[list[str]] = Noneable(ListValidator(EnumValidator(OpenDataSwissOperationTimeDaysOfWeek)))
+
+
+@validataclass
+class OpenDataSwissPropertiesInput:
+    operator: str = StringValidator()
+    displayName: str = StringValidator()
+    address: Optional[OpenDataSwissAddressInput] = Noneable(DataclassValidator(OpenDataSwissAddressInput))
+    capacities: list[OpenDataSwissCapacitiesInput] = ListValidator(DataclassValidator(OpenDataSwissCapacitiesInput))
+    additionalInformationForCustomers: Optional[OpenDataSwissAdditionalInformationInput] = Noneable(
+        DataclassValidator(OpenDataSwissAdditionalInformationInput)
+    )
+    parkingFacilityCategory: str = StringValidator()
```

Comment:
In my opinion, this can be an `EnumValidator`, or do you see any other value then `CAR`?